docusaurus/docs-internals/editor-controls/data-controls/iconPicker.md

```diff
@@ -0,0 +1,331 @@
+---
+toc_max_heading_level: 4
+---
+# Icon Picker
+
+The `iconPicker` control represents a list of icons available for selection.
```

Comment:
review description because this control it's actually a radio and return the choice value:
 
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/0de0c1f7-49ad-4790-a4b1-32604efe1f29)
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/b112e00d-dbd4-4539-bdf2-19ca59ca1a07)